src/renderer/pages/picbeds/index.vue

```diff
@@ -58,15 +54,36 @@ export default class extends Vue {
     // @ts-ignore
     const result = await this.$refs.configForm.validate()
     if (result !== false) {
-      this.saveConfig({
-        [`picBed.${this.type}`]: trimValues(result)
-      })
+      const configListConfigPath = `uploader.${this.type}.configList`
+      const configList = await this.getConfig<IStringKeyMap[]>(configListConfigPath)
+      // Finds the specified item from the config array and modifies it
+      const existItem = configList?.find(item => item._id === result._id)
+      // edit
+      if (existItem) {
+        Object.assign(existItem, result, {
+          _updatedAt: Date.now()
+        })
+      } else { // add new
+        configList?.push(completeUploaderMetaConfig(result))
```

Comment:
should also add `trimValues` for result
![image](https://user-images.githubusercontent.com/12621342/203200648-24453504-404e-45dc-af73-fbce4bbeea0d.png)